Harvest.Web/ClientApp/src/Projects/ProjectTable.tsx

```diff
@@ -7,6 +7,7 @@ import { ReactTable } from "../Shared/ReactTable";
 import { ReactTableUtil } from "../Shared/TableUtil";
 import { Project } from "../types";
 import { formatCurrency } from "../Util/NumberFormatting";
+import { convertCamelCase } from "../Util/ConvertCamelCase";
```

Comment:
I'm gonna nitpick a little here -- you have a non-default export called `convertCamelCase` from a file called `ConvertCamelCase`.  What other functions are you expecting to export?  I think you have two good choices here -- 1) would be change the file name to something more generic, like `StringFormatting.ts`, or (b) make the export default, since the file only does one thing (but then if you add more you'll have to change it).

Reply:
I will go with choice 1 just in case we need other string manipulation functions later on.